File: Work1.py
import random
randomNumber = random.randint(1, 100)
correct = False
print(randomNumber)

for i in range(3):
    userInput = int(input())

    if userInput > randomNumber:
        print("Too high, try again")
    elif userInput < randomNumber:
        print("Too low, try again")
    else:
        print("Your answer is correct")
        correct = True
        break

if not correct:
    print("Out of attempts")

File: Work1.java
import java.util.Random;
import java.util.Scanner;

class Work1 {
    public static void main(String[] args) {

        Random random = new Random();
        int randomNumber = random.nextInt(100) + 1;

        Scanner input = new Scanner(System.in);

        for (int i = 0; i < 3; i++) {
            int userInput = input.nextInt();

            if (userInput > randomNumber) {
                System.out.println("Too high, try again");
            } else if (userInput < randomNumber) {
                System.out.println("Too low, try again");
            } else {
                System.out.println("Your answer is correct");
                return;
            }
        }

        System.out.println("Out of attempts");
    }
}